examples/flex-flows/unify-ai/flex-flow-quickstart-unify-ai.ipynb

```diff
@@ -0,0 +1,932 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>C Hello World!</td>\n",
+       "      <td>1</td>\n",
+       "      <td>llama-3.1-8b-chat</td>\n",
+       "      <td>together-ai</td>\n",
+       "      <td>```c\\n#include &lt;stdio.h&gt;\\n\\nint main() {\\n    ...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>C# Hello World!</td>\n",
+       "      <td>2</td>\n",
+       "      <td>llama-3.1-8b-chat</td>\n",
+       "      <td>together-ai</td>\n",
+       "      <td>```csharp\\nusing System;\\n\\nclass HelloWorld \\...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "           inputs.text  inputs.line_number  inputs.model_name  \\\n",
+       "0  Python Hello World!                   0  llama-3.1-8b-chat   \n",
+       "1       C Hello World!                   1  llama-3.1-8b-chat   \n",
+       "2      C# Hello World!                   2  llama-3.1-8b-chat   \n",
+       "\n",
+       "  inputs.provider_name                                     outputs.output  \n",
+       "0          together-ai  ```\\n# Print Hello, World! to the console\\npri...  \n",
+       "1          together-ai  ```c\\n#include <stdio.h>\\n\\nint main() {\\n    ...  \n",
+       "2          together-ai  ```csharp\\nusing System;\\n\\nclass HelloWorld \\...  "
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "details = pf.get_details(base_run)\n",
+    "details.head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Evaluate your flow\n",
+    "Then you can use an evaluation method to evaluate your flow. The evaluation methods are also flows which usually using LLM assert the produced output matches certain expectation. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run evaluation on the previous batch run\n",
+    "The **base_run** is the batch run we completed in step 2 above, for web-classification flow with \"data.jsonl\" as input."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Prompt flow service has started...\n",
+      "You can view the traces in local from http://127.0.0.1:23333/v1.0/ui/traces/?#run=unify_llm_tool_20240804_202110_069589\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "[2024-08-04 20:21:10 +0000][promptflow._sdk._orchestrator.run_submitter][INFO] - Submitting run unify_llm_tool_20240804_202110_069589, log path: /root/.promptflow/.runs/unify_llm_tool_20240804_202110_069589/logs.txt\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
```

Comment:
may clean the outptus of the notebook before check in

Reply:
cleaned up outputs.